src/coreclr/vm/JitQCallHelpers.h

```diff
@@ -22,4 +22,71 @@ extern "C" void * QCALLTYPE ResolveVirtualFunctionPointer(QCall::ObjectHandleOnS
 extern "C" CORINFO_GENERIC_HANDLE QCALLTYPE GenericHandleWorker(MethodDesc * pMD, MethodTable * pMT, LPVOID signature, DWORD dictionaryIndexAndSlot, Module* pModule);
 extern "C" void QCALLTYPE InitClassHelper(MethodTable* pMT);
 
+#ifdef TARGET_32BIT
+extern "C" int32_t QCALLTYPE Math_ActualDivisionInt(int32_t i, int32_t j)
```

Comment:
The implementation should not be in a header file.

Reply:
I didn't know it applied to simple methods as well; there are tons of examples to the contrary in the vm headers. ;)

Agreed, this was to quickly address the previous feedback, still in draft since 2 out of 113 conformance cases failed on linux-arm https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-109087-merge-97337955ae1a475f89/IL_Conformance/1/console.2ba26364.log?helixlogtype=result

I'm also looking into JIT-only alternative approach.

Reply:
I do not think any QCALL implementations live in header files.